### DISCUSSION TITLE
Tips/tricks for optimising results... Video?

### DISCUSSION THREAD

**Discussion by StarGeezerPhil** [aaronwmorris/indi-allsky#626] [General]
Didn't know if anyone might be able to share any tricks for getting the most out of their setup?

I've seen some really impressive allsky **videos** of meteors, but presumably this isn't something we could do with INDI-allsky as the focus is on exposure time, individual frames, stacking (* in progress) and then stitching timelapses together. 

Is there an option or any point in looking at this @aaronwmorris, I presume there isn't any feasible way to achieve both as the camera will be busy - or would there be a technique to output frames from a video into a (simulated) long exposure for each frame/image (that we store in the current way)?

I'm using the IMX378 but currently assessing the benefit/pro/con of upping gain for reduced exposure times and more frequent imaging .vs. less images, longer exposures. Sadly, weather against me here in the UK recently so not getting many nights to compare results.

Interested to hear any tips/tricks from the community here.

**Comment by aaronwmorris** [comment]
The design of indi-allsky is focused exclusively on timelapse photography.  I have some interesting in looking at video acquisition, but that would be a completely separate project from indi-allsky.

For the IMX378 and IMX477, I have yet to deploy one looking at the sky.  I have both sensors, but I have only used them as dev cameras.  My next camera will be one of these two.

I do need to find a good stretching algorithm.  The RAW images from the IMX477/378 are rather.... flat.  You may try experimenting with binning.

**Comment by bluthen** [comment]
> I do need to find a good stretching algorithm. The RAW images from the IMX477/378 are rather.... flat. You may try experimenting with binning.

In anycase I've seen good results using:
https://github.com/LCOGT/auto_stretch

I also use greyworld auto white balance.

Sorry if you already tried it, I tried to grep through your code to see what you currently do.
